demo/course.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version='1.0' encoding='utf-8'?>
+<odoo>
+        <record id="react_course" model="open.academy.ourse">
```

Comment:
```suggestion
        <record id="open_academy_course_react_demo" model="open.academy.course">
```

See [guidelines](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#254demo-records) to name and xml_id and apply it to all the ids in this file please.